src/endpoints/transfers/transfer.controller.ts

```diff
@@ -162,6 +165,8 @@ export class TransferController {
     @Query('before', ParseIntPipe) before?: number,
     @Query('after', ParseIntPipe) after?: number,
     @Query('round', ParseIntPipe) round?: number,
+    @Query('relayer', ParseAddressPipe) relayer?: string,
+    @Query('isRelayed', new ParseBoolPipe) isRelayed?: boolean,
```

Comment:
you can remove the `new` keyword if you don't add any params to ParseBoolPipe

Reply:
fixed